packages/webpack-loader/src/__tests__/extract-plugin.test.tsx

```diff
@@ -13,34 +13,44 @@ describe('CompiledExtractPlugin', () => {
       mode: 'production',
     });
 
-  it('throws when the plugin is not configured', async () => {
-    const errors = await bundle(join(fixturesPath, 'local-styles.tsx'), {
-      disableExtractPlugin: true,
-    }).catch((err) => err);
-
-    expect(errors).toEqual([
-      expect.objectContaining({
-        message: expect.stringContaining("You forgot to add the 'CompiledExtractPlugin' plugin"),
-      }),
-    ]);
-  });
-
-  it('extracts local styles', async () => {
-    const actual = await bundle(join(fixturesPath, 'local-styles.tsx'));
-
-    expect(actual[assetName]).toMatchInlineSnapshot(`
+  it.concurrent(
+    'throws when the plugin is not configured',
+    async () => {
+      const errors = await bundle(join(fixturesPath, 'local-styles.tsx'), {
+        disableExtractPlugin: true,
+      }).catch((err) => err);
+
+      expect(errors).toEqual([
+        expect.objectContaining({
+          message: expect.stringContaining("You forgot to add the 'CompiledExtractPlugin' plugin"),
+        }),
+      ]);
+    },
+    10000
```

Comment:
Had to bump the timeout as some tests were failing locally/in CI. It might be because webpack has to do more work now transpiling on the fly instead of going straight to source.